docs/nGraph_DG/nGraphTransformation.md

```diff
@@ -32,7 +33,7 @@ But if some transformation parts can potentially be reused in other transformati
 To decide where to store your transformation code, please follow these rules:
 
 1. If it is a plugin-specific transformation and cannot be reused by other plugins, keep source code inside plugin.
-2. If this transformation relates to the OpSetXToOpSetY conversion or it is common optimization, keep sources inside the transformation library.
+2. If this transformation relates to opset operation conversion or optimization then keep sources inside the transformation library.
 
 After you decide where to store your transformation code, you can start developing your own nGraph transformation.
```

Comment:
Could you update a paragraph about ngraph function and add that nGraph function contains pointers to  `Parameter`, `Result` and `Sink`?

Reply:
Actually I want to divide this doc into several parts. In this PR I juts update transformations API related topics.